examples/__testfixtures__/wl-components/account-orders/styled.base.output.ts

```diff
@@ -255,8 +255,7 @@ export const RecentOrderTitle = Header.withConfig({
   marginX: 0,
   marginY: '$8',
   marginBottom: '$5',
-  // TODO: This should not be like this
-  color: '__2substitution__ !important',
+  color: '__legacyToken.text-default',
```

Comment:
nice!

Reply:
Yeah, i was impressed it resolve ddown to this so directly!